depthimage_to_pointcloud2/src/depthimage_to_pointcloud2_node.cpp

```diff
@@ -37,7 +38,7 @@ static void depthCb(const sensor_msgs::msg::Image::SharedPtr image)
 
   if (nullptr == g_cam_info) {
     // we haven't gotten the camera info yet, so just drop until we do
-    fprintf(stderr, "No camera info, skipping point cloud conversion\n");
+    RCUTILS_LOG_WARN("No camera info, skipping point cloud conversion")
```

Comment:
I think this (and the next one below) should be ERROR and not WARN given that they result in the function returning without publishing the pointcloud

Reply:
True, but for this one in particular, it happens regularly at startup time.  Sometimes this node comes up before both the depth image and the camera info have been published, so you see this message those times.  At that point it is not an error, but really just informational.  I'm not sure what we should do with that.